validator/slashing-protection/local/attesting-history/history.go

```diff
@@ -19,8 +19,18 @@ const (
 	signingRootSize        = 32
 	historySize            = targetSize + sourceSize + signingRootSize
 	minimalSize            = latestEpochWrittenSize
+	// Key prefix to minimal attestation source epoch in attestation bucket.
+	minimalAttestationSourceEpochKeyPrefix = "minimal-attestation-source-epoch"
```

Comment:
We are mixing up bolt logic with this package that knows nothing about bolt. Perhaps move to bolt itself or refactor into something more general